frontend/server/src/Controllers/Course.php

```diff
@@ -708,7 +708,7 @@ public static function apiClone(\OmegaUp\Request $r): array {
                         false, // visbility mode validation no needed when it is a clone
                         100,
                         null,
-                        1
+                        intval($problem['order'])
```

Comment:
el hecho de que le tenemos que hacer un cast explícito a algo que debería ser entero _suena_ a bug. qué pasa si se quita?

```suggestion
                        $problem['order']
```